### ISSUE TITLE
Tilde char in fields and document

### ISSUE THREAD

**Issue by christianmahnke** [LibreCat/LibreCat#246]
During our export and import test we noticed that sometimes a tilde sign appears in fields or even as a whole document, two examples:
```
---
_id: 3145381
_version: 1
abstract:
  - lang: eng
    text: "Admired by philosophers such as Schopenhauer, Nietzsche, Kierkegaard, Freud, Benjamin, and Wittgenstein, Georg Christoph Lichtenberg (1742-1799) is known to the English-speaking world mostly as a satirist. An eminent experimental physicist and mathematician, Lichtenberg was knowledgeable about the philosophical views of his time, and interested in uncovering the philosophical commitments that underlie our common beliefs. In his notebooks (which he called his Waste Books) he often reflects on, challenges, and critiques these philosophical commitments and the dominant views of the Enlightenment, German idealism, and British empiricism. This scholarly collection of Lichtenberg's philosophical aphorisms contains hundreds of trenchant observations drawn from these notebooks, many of which have been translated into English here for the first time. It also includes a historical and philosophical introduction to his writings, situating him in the history of philosophy and ideas, and is supplemented with a chronology, suggestions for further reading, and extensive introductory and textual notes explaining his references."
citation:
  ama: 'Tester S, ed. Georg Christoph Lichtenberg. Philosophical Writings. Albany: State University of New York Press; 2012.'
  apa: 'Tester, S. (Ed.). (2012). Georg Christoph Lichtenberg. Philosophical Writings. Albany: State University of New York Press.'
  chicago: 'Tester, Steven, ed. Georg Christoph Lichtenberg. Philosophical Writings.  	SUNY Series in Contemporary Continental Philosophy . Albany: State University of New York Press, 2012.'
  ieee: 'S. Tester, Ed., Georg Christoph Lichtenberg. Philosophical Writings. Albany: State University of New York Press, 2012.'
  mla: 'Tester, Steven, ed. Georg Christoph Lichtenberg. Philosophical Writings. Albany: State University of New York Press, 2012. Print.  	SUNY Series in Contemporary Continental Philosophy .'
  short: 'S. Tester, ed., Georg Christoph Lichtenberg. Philosophical Writings, State University of New York Press, Albany, 2012.'
creator:
  id: 101
  login: claudia_gro
date_created: 2017-03-22T14:45:15Z
date_updated: 2017-03-22T14:45:15Z
editor:
  - first_name: Steven
    full_name: 'Tester, Steven'
    id: stester
    last_name: Tester
language:
  - iso: eng
message: aufklaerung
page: 223
place: Albany
publication_identifier:
  isbn:
    - ~
publisher: State University of New York Press
series_title: ' 	SUNY series in contemporary Continental philosophy '
status: public
title: Georg Christoph Lichtenberg. Philosophical Writings
type: book_editor
user_id: 101
year: 2012
```
(here it is the field `isbn`)

Or for a whole document:
```
--- ~
```
Error message is
```
ERROR: not a valid publication
/language: Expected array - got string.
```

**Comment by christianmahnke** [comment]
The case of the malformed document is quite easy to reproduce on 0.5.4:

1. Create a new publication but don't save it.
2. Delete the document which has never been save in the first place 

You get an error message:
```
{
   "error" : "Warning caught during route execution: Use of uninitialized value in string eq at /opt/local/lib/LibreCat/App/Helper.pm line 398.\n",
   "exception" : "Warning caught during route execution: Use of uninitialized value in string eq at /opt/local/lib/LibreCat/App/Helper.pm line 398.\n"
}
```

You have something with an if you open the internal view with the id that should have been used, it looks like this:
```
--- ~



```
(yes, the line breaks seem to be in there)

**Comment by vpeil** [comment]
`~` in YAML means empty.

**Comment by christianmahnke** [comment]
The part with the malformed ISBN might be something related to #182. But then again, this record was created after this issue was closed.

**Comment by phochste** [comment]
Fixed with `ad2f308` you need pull in the changed and generate forms.
